src/shared/HtmlReady.js

```diff
@@ -110,7 +110,7 @@ export default function(html, { mutate = true, hideImages = false } = {}) {
         };
     } catch (error) {
         // xmldom error is bad
```

Comment:
Good idea to log it, not elegant but a a simple `console.log('RENDER ERROR: '+error.message)` would let us keep track of it server-side at least

Reply:
i just set it up to also log the problematic html... this may or may not be a good idea ;)